doc/cephfs/administration.rst

```diff
@@ -42,6 +42,34 @@ creation of multiple filesystems use ``ceph fs flag set enable_multiple true``.
     fs rm_data_pool <filesystem name> <pool name/id>
 
 
+Settings
+--------
+
+::
+
+    fs set <fs name> max_file_size <size in bytes>
+
+CephFS has a configurable maximum file size, and it's 1TB by default.
+You may wish to set this limit higher if you expect to store large files
+in CephFS. It is a 64-bit field.
+
+CephFS enforces this limit at the point of appending to files or
+setting their size. It does not affect how anything is stored.
+
+When users create a file of an enormous size (without necessarily
+writing any data to it), some operations (such as deletes) cause the MDS
+to have to do a ridiculously large number of operations to check if any
```

Comment:
I don't think you need to point out that the number is ridiculously large. s/ridiculously//g reads OK to me